nameko_amqp_retry/backoff.py

```diff
@@ -44,7 +48,7 @@ def get_next_schedule_item(cls, index):
             item = cls.schedule[index]
         return item
 
-    def get_next_expiration(self, message, backoff_exchange_name):
+    def calculate_next_expiration(self, message, backoff_exchange_name):
```

Comment:
The name `calculate_next_expiration()` does not suggest any side effect, looks like it only calculates and returns.

Maybe `set_next_expiration()` would be a better name?

Or `next_expiration()`, or just `next()` setting and also returning the next expiration would also do? It then hints the side effect inside as looks similar to the iterator `next` function.